core/business.go

```diff
@@ -186,6 +227,13 @@ func (w Worker) Start() {
 						w.JobChannel <- job
 					}
 				} else {
+					// If job type is transfer and we get success then delete that request from heap. Make the status in DB to finished.
+					if job.Action == "transfer" {
+						job.Action = "delete"
+						job.TransferRequest.Delay = 0
+						job.TransferRequest.Status = "success"
+						w.JobChannel <- job
```

Comment:
If transfer job successfully passed make one other job to delete request from heap and update the status in DB.